### PR TITLE
ON-27707 # Added lookup documentation examples

### PR DESCRIPTION
**Pull request by jdawg093**

### CODE REVIEW

File: docs/examples/calling-external-api-lookup.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+Your form should now look something like this: 
+
+![A gif showing the form layout for the example](../../docs/pics/ExternalAPIForm.gif)
+
+Please make sure that your element names match here, as you will need to have the correct elements for the API code! With the form out of the way, we can now write the API Code!
+
+## Writing The API Code
+
+You are able to grab the code that is about to be written [by clicking here](../../examples/api/lookup-data-transfer)
```
Comment (kizaonline) [inline docs/examples/calling-external-api-lookup.md:48]:
```suggestion
You are able to grab the code that is about to be written [by clicking here](../../examples/api/lookup/src/data-transfer.js)
```

File: docs/examples/calling-external-api-lookup.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+Now that we have covered that, let us write the last part of the lookup!
+
+### Format and return that data from the lookup
+
+We have the data we require, we need to now return it for the user. We can do this by writing this following code:
+```js
+return res.setStatusCode(200).setPayload({
+  activity: activityReturned.activity,
+  participants: +activityReturned.participants,
```
Comment (kizaonline) [inline docs/examples/calling-external-api-lookup.md:201]:
```suggestion
  participants: activityReturned.participants,
```

File: docs/examples/calling-external-api-lookup.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  price: +activityReturned.price,
```
Comment (kizaonline) [inline docs/examples/calling-external-api-lookup.md:202]:
```suggestion
  price: activityReturned.price,
```

File: docs/examples/calling-external-api-lookup.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+  link: activityReturned.link,
+  key: activityReturned.key,
+  accessibility: +activityReturned.accessibility,
```
Comment (kizaonline) [inline docs/examples/calling-external-api-lookup.md:205]:
```suggestion
  accessibility: activityReturned.accessibility,
```

File: docs/examples/calling-external-api-lookup.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+|-- project-root
+|   |-- .blinkmrc.json
+|   |-- src
+|   |   |-- routes
+|   |   |   |-- fill-data-from-api.js
+```
+
+## Using the new Code on the Console
+
+Now that we have written all of this code, we can now deploy it to the Console. If you have not hosted an API via OneBlink before, you can [follow this article here to understand how](../api/hosting-api.md).
```
Comment (kizaonline) [inline docs/examples/calling-external-api-lookup.md:235]: Dont think that link exists

**Review by kizaonline — CHANGES_REQUESTED** [review]

File: docs/examples/transferring-data-pages.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+module.exports.post = function (req, res){
+
+}
+```
+
+We have called the function `post` and given it the parmameters of `req` and `res`. These will be objects that we will utilise during the function.
+
+You can read what is included in the `Request` and `Response` objects [in our article featured here](../api/handlers.md), under the `Request` and `Response` headings.
+
+Please keep in mind, that there is another way of writing these functions involving ES Modules, you can see how to update the function to use that method [here](./upgrading-to-es-modules.md)
```
Comment (jdawg093) [inline docs/examples/transferring-data-pages.md:51]: I think this one needs to be updated too

File: docs/examples/calling-external-api-lookup.md (hunk 5 above)
Comment (jdawg093) [reply docs/examples/calling-external-api-lookup.md:235]: The link is to a file that is in the other MR, so when they are both merged, the link should work :) I initially wrote this MR while still in the other branch and separated them for when I wanted to create their MRs.

Comment (kizaonline) [reply docs/examples/calling-external-api-lookup.md:235]: This PR should probably target that other branch then?

Comment (jdawg093) [reply docs/examples/calling-external-api-lookup.md:235]: Targetting the other branch now.

**Review by kizaonline — APPROVED** [review]

**Pull request merged**